release.md

```diff
@@ -42,21 +42,33 @@ release.
 
 ### Preparation
 
-Make sure you're on an up-to-date master branch and in cirq's root directory.
+System requirements: Linux, python3.7
+
+For MINOR / MAJOR release: Make sure you're on an up-to-date master branch and 
+in cirq's root directory.
 
 ```bash
 git checkout master
 git pull origin master  # or upstream master
 git status  # should be no pending changes
 ```
 
+For PATCH update: Make sure you're in the last version 
+
+```bash
+git fetch origin # or upstream - to fetch all tags
+export LATEST_TAG=$(git tag | tail -n 1) # check with echo $LATEST_TAG 
```

Comment:
This seems risky, since it assumes that the last tag is what we want.  What if we patch both 0.8.2 and 0.7.1? Or create a tag for a non-release reason?

At the very least, I would change the echo to an actual command, like echo "You are in branch $LATEST_TAG".

Reply:
I agree that this is risky - especially after 1.0 this might be a real situation. I am just going to remove this part and introduce the variable for "$VERSION_TO_PATCH".